src/components/Display.js

```diff
@@ -0,0 +1,6 @@
+import Calculator from './Calculator';
+
+function Display() {
```

Comment:
- You have done a great job💪 creating the Display component. However🙆‍♀️, this component is supposed to be used inside the Calculator component, and you have done vice versa. 🙏Please use the Display component inside the Calculator component so that it becomes a child🧒 component, meeting the project requirements. Kindly check the below example

```jsx
function Calculator() {
  return (
    <Display />
  );
}
```